static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -28,8 +33,27 @@ const setupPrebid: () => Promise<void> = () =>
             getPageTargeting();
             prebid.initialise(window);
         }
-        return Promise.resolve();
-    });
+    })
+}
+
+const setupPrebid: () => Promise<void> = () => {
+    let canRun: boolean = true;
+    const isInTcfv2EnforcementVariant = isInVariantSynchronous(googletagPrebidEnforcement, 'variant')
+    if (shouldUseSourcepointCmp()) {
+        onConsentChange(state => {
+            // Only TCFv2 mode can prevent running Prebid
+            if (state.tcfv2) canRun = state.tcfv2.vendorConsents[SOURCEPOINT_ID];
+            if (canRun && isInTcfv2EnforcementVariant) {
```

Comment:
I think the same logic of only having `loadPrebid` once can be applied here.

```suggestion
            if (state.tcfv2) canRun = isInTcfv2EnforcementVariant ? state.tcfv2.vendorConsents[SOURCEPOINT_ID] : true;
            if (canRun) {
```